### ISSUE TITLE
Add option to get queries along with pack in fleetctl get -p

### ISSUE THREAD

**Issue by benbasscom** [kolide/fleet#1975]
### What version of `fleet` are you using (`fleet version --full`)?
{
  "version": "2.0.1",
  "branch": "master",
  "revision": "d01727dff26683ba29424fbdb7e58e403ab6210d",
  "go_version": "go1.11.2",
  "build_date": "2018-11-26T19:21:48Z",
  "build_user": "zwass"
}
### What operating system are you using?
RHEL 7.6 server, macOS 14.2 client/admin

### What did you do?
fleetctl get -p "my awesome pack"

### What did you expect to see?

Yaml output of the entire pack, including the queries, enabling applying the pack to generate a full pack in fleet.  This would match the output of converting a json pack to yaml.

### What did you see instead?
Yaml output of the pack without the queries at the bottom, just the names and schedules of the queries.

**Comment by zwass** [comment]
Technically a pack definition does not include the query SQL (because queries may be shared across multiple packs), but I can see how this would be a useful feature.

**Comment by TonyLeeVT** [comment]
It appears that the request is to add an option that does the following:
1)  First retrieve the pack
2)  For each query listed in the pack, retrieve the query

This provides a bit of an export feature from fleetctl which is also what I am seeking.  This is useful for backing up a fleet's packs and queries as well as making it easier to package and share packs and queries with others.  Thanks for your continued support.

**Comment by icebawkz** [comment]
A dirty workaround for this can be achieved using pipes:

`fleetctl get pack unwanted-chrome-extensions && fleetctl get pack unwanted-chrome-extensions | grep -Po "(?<=name: )[^\n]*" | xargs -n1 -I {} sh -c "fleetctl get query {}"`

```
apiVersion: v1
kind: pack
spec:
  id: 4
  name: unwanted-chrome-extensions
  queries:
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: TouchVPN
    query: TouchVPN
  - description: (https://www.bleepingcomputer.com/news/security/chrome-extension-with-over-one-million-users-hijacked-to-serve-adware/)
    interval: 3600
    name: WebDeveloper
    query: WebDeveloper
  - description: (https://www.virustotal.com/#/file/5cab0821f597100dc1170bfef704d8cebaf67743e9d509e83b0b208eb630d992/detection)
    interval: 3600
    name: MacOSInstallCore
    query: MacOSInstallCore
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: Chrometana
    query: Chrometana
  - description: (https://www.bleepingcomputer.com/news/security/copyfish-chrome-extension-hijacked-to-show-adware/)
    interval: 3600
    name: CopyFish
    query: CopyFish
  - description: (http://adios-hola.org)
    interval: 3600
    name: HolaVPN
    query: HolaVPN
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: InfinityNewTab
    query: InfinityNewTab
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: SocialFixer
    query: SocialFixer
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: BetternetVPN
    query: BetternetVPN
  - description: (https://www.reddit.com/r/chrome/comments/6htzan/psawarning_giphy_extension_6172017_is_now_malware/)
    interval: 3600
    name: Giphy
    query: Giphy
  - description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
    interval: 3600
    name: WebPaint
    query: WebPaint
  targets:
    labels:
    - All Hosts
The resource was not found
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: TouchVPN
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='bihmplhobchoageeokmgbdihknkjbknd';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/chrome-extension-with-over-one-million-users-hijacked-to-serve-adware/)
  name: WebDeveloper
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='bfbameneiokkgbdmiekhjnmfkcnldhhm';
apiVersion: v1
kind: query
spec:
  description: (https://www.virustotal.com/#/file/5cab0821f597100dc1170bfef704d8cebaf67743e9d509e83b0b208eb630d992/detection)
  name: MacOSInstallCore
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='hinehnlkkmckjblijjpbpamhljokoohh';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: Chrometana
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='kaicbfmipfpfpjmlbpejaoaflfdnabnc';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/copyfish-chrome-extension-hijacked-to-show-adware/)
  name: CopyFish
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='eenjdnjldapjajjofmldgmkjaienebbj';
apiVersion: v1
kind: query
spec:
  description: (http://adios-hola.org)
  name: HolaVPN
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='gkojfkhlekighikafcpjkiklfbnlmeio';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: InfinityNewTab
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='dbfmnekepjoapopniengjbcpnbljalfg';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: SocialFixer
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='ifmhoabcaeehkljcfclfiieohkohdgbb';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: BetternetVPN
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='gjknjjomckknofjidppipffbpoekiipm';
apiVersion: v1
kind: query
spec:
  description: (https://www.reddit.com/r/chrome/comments/6htzan/psawarning_giphy_extension_6172017_is_now_malware/)
  name: Giphy
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='kaicbfmipfpfpjmlbpejaoaflfdnabnc';
apiVersion: v1
kind: query
spec:
  description: (https://www.bleepingcomputer.com/news/security/eight-chrome-extensions-hijacked-to-deliver-malicious-code-to-4-8-million-users/)
  name: WebPaint
  query: SELECT * FROM users JOIN chrome_extensions USING (uid) WHERE identifier='emeokgokialpjadjaoeiplmnkjoaegng';
```

**Comment by TonyLeeVT** [comment]
Thanks Icebawkz.  I ended up writing a wrapper called fleet_helper that does quite a bit more.  Documentation and code found here:

[http://securitysynapse.blogspot.com/2019/06/osquery-part-vi-controlling-fleetctl.html](http://securitysynapse.blogspot.com/2019/06/osquery-part-vi-controlling-fleetctl.html)

```
Usage:  ./fleet_helper.sh <function>

Note: All functions output to stdout.  Redirect to .yaml files as needed.

Possible functions:
 listpacks - Lists pack names only within the Kolide Fleet instance
 listqueries - Lists query names only within the Kolide Fleet instance
 exportpacks - Exports all packs in yaml format
 exportqueries - Exports all queries in yaml format
 exportpack <pack_name> - Exports a specified pack in yaml format
 exportquery <query_name> - Exports a specified query in yaml format
 exportpackquery <pack_name> - Exports a pack and all associated queries in yaml format
 exportall - Warning!  This exports all packs and all queries in yaml format
```

**Comment by zwass** [comment]
Thanks for the interest everyone. We will get to this, but I'm glad folks have built some scripts that suffice for now.

**Comment by zwass** [comment]
Closed by #2092
